sdk/examples/intkey_python/sawtooth_intkey/client_cli/create_batch.py

```diff
@@ -59,7 +59,8 @@ def sha512(self):
         return self._sha512
 
 
-def create_intkey_transaction(verb, name, value, private_key, public_key):
+def create_intkey_transaction(verb, name, value, deps,
```

Comment:
Since you're modifying the function signature, could you add some doc string here?